docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -612,3 +633,28 @@ import (
 
 - the `conditions` package provides access to utils for managing `metav1.Conditions` in `status.conditions`
 - the `patch` package provides access to utils for patching objects in this phase
+
+Important!
+- Please pay special attention to use `sigs.k8s.io/cluster-api/util/patch` import everywhere,
+  because if by mistake you use `sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch` in some files this won't lead to compile errors
+  but conditions are going to be dropped at runtime.
+
+## Annex
+
+### Imports for conditions and patch helper utils
+
+In order to help users transition away from CAPI conditions type, CAPI v1.11 supports
+different versions of conditions and patch helper utils.
+
+Following table should help in using the right version
+
+| Field to change                                                       | Import for condition util                                                                | Import for patch helper                                              |
+|-----------------------------------------------------------------------|------------------------------------------------------------------------------------------|----------------------------------------------------------------------|
+| `status.conditions` of type `clusterv1beta1.Conditions`               | v1beta1conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"           | v1beta1patch "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch" |
+| `status.v1beta2.conditions` of type `[]metav1.Conditions`             | v1beta2conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions/v1beta2"   | v1beta1patch "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch" |
+| `status.deprecated.v1beta1.conditions` of type `clusterv1.Conditions` | deprecatedv1beta1conditions "sigs.k8s.io/cluster-api/util/conditions/deprecated/v1beta1" | "sigs.k8s.io/cluster-api/util/patch"                                 |
+| `status.conditions` of type `[]metav1.Conditions`                     | "sigs.k8s.io/cluster-api/util/conditions"                                                | "sigs.k8s.io/cluster-api/util/patch"                                 |
+
+Important!
+- Please pay special attention to use the correct patch helper import everywhere, because if by mistake you use a wrong
+  onw in some files this won't lead to compile errors but conditions are going to be dropped at runtime.
```

Comment:
```suggestion
- Please pay special attention to use the correct patch helper import everywhere, because using a wrong
  one could in some cases lead to dropping conditions at runtime while not having compile errors.  

```